utils/open-cas-linux.spec.base

```diff
@@ -87,20 +87,20 @@ fi
 
 
 %post modules_%{kver_filename}
-ls /lib/modules/%{kver}/extra/cas_disk.ko | weak-modules --no-initramfs --add-modules
-ls /lib/modules/%{kver}/extra/cas_cache.ko | weak-modules --no-initramfs --add-modules
+modules=( $(find /lib/modules/%{kver}/extra/ -name "cas_*.ko") )
+printf "%s\n" "${modules[@]}" | weak-modules --no-initramfs --add-modules
 depmod
 
 %preun modules_%{kver_filename}
 if [ $1 -eq 0 ]; then
-    rmmod cas_cache
-    rmmod cas_disk
-    ls /lib/modules/%{kver}/extra/cas_disk.ko | weak-modules --no-initramfs --remove-modules
-    ls /lib/modules/%{kver}/extra/cas_cache.ko | weak-modules --no-initramfs --remove-modules
+    find /lib/modules/%{kver}/extra/ -name "cas_*.ko" >/var/run/rpm-open-cas-linux-modules
 fi
 
 %postun modules_%{kver_filename}
 if [ $1 -eq 0 ]; then
+    modules=( $(cat /var/run/rpm-open-cas-linux-modules) )
+    rm -f /var/run/rpm-open-cas-linux-modules
+    printf "%s\n" "${modules[@]}" | weak-modules --no-initramfs --remove-modules
```

Comment:
I think you can replace all this with:
( modinfo -F filename cas_disk;  modinfo -F filename cas_cache) | weak-modules --no-initramfs --remove-modules